backend/EasySpeak.Core/EasySpeak.Core.BLL/Services/LessonsService.cs

```diff
@@ -45,4 +45,24 @@ public async Task<ICollection<LessonDto>> GetAllLessonsAsync(FiltersRequest filt
 
         return lessonDtos;
     }
-}
+
+    public async Task<ICollection<DayCardDto>?> GetDayCardsOfWeekAsync(RequestDayCardDto requestDto)
+    {
+        var dayCards = await _context.Lessons
+            .Where(c => c.StartAt.Date >= requestDto.Date.Date
+                        && c.StartAt.Date <= requestDto.Date.AddDays(6).Date
```

Comment:
Can you make sure that the date you use will always be Monday? Independent of what you get in request
Just get the first Monday before that date

Reply:
fixed